tests/tdastro/sources/test_snia.py

```diff
@@ -129,11 +129,25 @@ def test_snia_end2end(
 
     url_base = "https://raw.githubusercontent.com/lsst/throughputs/e70d1daf069e606caa3feb43eccc62ec21e0baf5/"
     passbands = PassbandGroup(
-        passbands=[
-            Passband("LSST", "g", table_url=f"{url_base}baseline/total_g.dat", units="nm"),
-            Passband("LSST", "r", table_url=f"{url_base}baseline/total_r.dat", units="nm"),
-        ]
+        passband_parameters=[
+            {
+                "survey": "LSST",
+                "filter_name": "u",
+                "table_url": f"{url_base}baseline/total_u.dat",
+                "units": "nm",
+                "force_download": True,
+            },
+            {
+                "survey": "LSST",
+                "filter_name": "r",
+                "table_url": f"{url_base}baseline/total_r.dat",
```

Comment:
How large are these files? Could we include them (or a suitable fake) in the git repository? Downloading files from a URL during a test increases the change the test will randomly fail. So we should try to avoid that as much as possible.